modules/installation-vsphere-machines.adoc

```diff
@@ -120,7 +119,7 @@ $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${
 +
 *** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
 *** Click *Edit Configuration*, and on the *Configuration Parameters* window, click *Add Configuration Params*. Define the following parameter names and values:
-**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type.
+**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type.  Note: For the bootstrap node, ignition must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window.  This is due to a restriction in the maximum size of data which may be provided in a vApp property.
```

Comment:
```suggestion
**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type. Note for the bootstrap node, the Ignition config file must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window. This is due to a restriction in the maximum size of data that can be provided in a vApp property.
```

Reply:
@rvanderp3 just some wordsmithing nits remaining for me, then it'll be good from my side. Thanks!

Reply:
No problem at all

Reply:
Done!

Reply:
Thank you!